Uno.Gallery/Views/SamplePages/SimpleOrientationSamplePage.xaml.cs

```diff
@@ -1,84 +1,98 @@
-﻿using System;
```

Comment:
What's the actual change that was done here? It seems to be that the whole file content was changed so it's not highlighting the specific change for the fix

Reply:
formatting change like this, while welcomed,
should be in its own PR or, the very least, its own commit

Reply:
It was not supposed to be a formatting change, some how Visual Studio Code made these changes. 

I am looking for a way to revert these space changes.

Reply:
Fixed!